src/Formio.js

```diff
@@ -530,7 +530,7 @@ class Formio {
     });
   }
 
-  uploadFile(storage, file, fileName, dir, progressCallback, url, options, fileKey, groupPermissions, groupId, uploadStartCallback) {
```

Comment:
We cannot change the signature of these methods without it being considered a "breaking change". Please do this in a way where the method signature is reverse compatible.